crates/storage/src/exec.rs

```diff
@@ -99,7 +99,7 @@ macro_rules! inst_ops {
             }
 
             #[async_trait::async_trait]
-            trait ShimTrait: Sync + Send + 'static {
+            pub(crate) trait ShimTrait: Sync + Send + 'static {
```

Comment:
Why does this need to be `pub(crate)`?

Reply:
Actually it doesn't have to be. Updated.